### PR TITLE
Feature/33 add new homepage

### PR DESCRIPTION
**Pull request by navins94**
- #33

### CODE REVIEW

File: frontend/src/pages/home/style.scss
Comment (wayangalihpratama) [inline frontend/src/pages/home/style.scss:1]: Hi @navins94 ,
What if we move the 'font-size' style into the global (App.scss) file? This way, we can have the same font size on every page and avoid redundantly declaring the style on each page.

**Review by wayangalihpratama — CHANGES_REQUESTED** [review]

File: frontend/src/pages/home/style.scss
Comment (wayangalihpratama) [reply frontend/src/pages/home/style.scss:1]: Also I have this warning when running this project
Screenshot from 2023-04-10 09-38-20

Comment (navins94) [reply frontend/src/pages/home/style.scss:1]: I will add font-size in globally

Comment (navins94) [reply frontend/src/pages/home/style.scss:1]: Need to check from where originalPromt error is coming. It is not coming on my local

**Pull request closed without merging**